### ISSUE TITLE
How can I get spectrum waveforms?

### ISSUE THREAD

**Issue by metcenger** [rutvik110/flutter_audio_waveforms#33]
I want to drow spectrum of signal, not a oscillogram. Is it possible to do it?

**Comment by rutvik110** [comment]
Could you please provide more info on exactly what you wanna achieve? any visual aid will be helpful.

**Comment by metcenger** [comment]
Sorry for delay with answer.
![image](https://user-images.githubusercontent.com/11522773/221345764-e41473fd-f061-4e62-a013-9a2c59a4183f.png)
That what I meant. 
Graph with frequency and amplitude.

**Comment by Jafar-Rezazadeh** [comment]
yes i need some thing like this too ,  get the current playing audio amplitude , i mean getting the exact amplitude or energy of the audio in that specific time.
for example : we have 3 min audio i want to get the exact amplitude of each 100 millisecound,
i think we could get it now but the problem is we are getting it as list<double> but i want to get the double while playing the audio

**Comment by xurshid20222** [comment]
> yes i need some thing like this too , get the current playing audio amplitude , i mean getting the exact amplitude or energy of the audio in that specific time. for example : we have 3 min audio i want to get the exact amplitude of each 100 millisecound, i think we could get it now but the problem is we are getting it as list but i want to get the double while playing the audio

me too
